apps/lib/engine_loader.c

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include "apps.h"
+
+#ifndef OPENSSL_NO_ENGINE
+
+/*
+ * Here is an STORE loader for ENGINE backed keys.  It relies on deprecated
+ * functions, and therefore need to have deprecation warnings suppressed.
+ * This file is not compiled at all in a '--api=3 no-deprecated' configuration.
+ */
+# define OPENSSL_SUPPRESS_DEPRECATED
+
+# include <stdarg.h>
+# include <string.h>
+# include <openssl/engine.h>
+# include <openssl/store.h>
+
+
```

Comment:
Here are two empty lines.
`check-format.pl apps/lib/engine_loader.c` just reports this and three more whitespace nits.

Reply:
Fixed